cesium/features/lomb_scargle.py

```diff
@@ -51,6 +52,14 @@ def lomb_scargle_model(
         Order of polynomial to fit to the data while
         fitting the dominant frequency. Defaults to 1.
 
+    freq_grid : dict or None, optional
+        Grid parameters to use. If None, then calculate the frequency
+        grid automatically. To supply the grid, keys
+        ["f0", "fmax"] are expected. "f0" is the smallest frequency
```

Comment:
Where do `df` and `numf` come from?

Reply:
they are either supplied by the user or calculated using lines 87 & 89.